docker/sui-indexer-alt/Dockerfile

```diff
@@ -0,0 +1,37 @@
+# Build application
+#
+# Copy in all crates, Cargo.toml and Cargo.lock unmodified,
+# and build the application.
+FROM rust:1.81-bullseye AS builder
+ARG PROFILE=release
+ARG GIT_REVISION
+ENV GIT_REVISION=$GIT_REVISION
+WORKDIR "$WORKDIR/sui"
+
+# sui-indexer needs postgres libpq5 and ca-certificates
+RUN apt update && apt install -y libpq5 ca-certificates libpq-dev postgresql
```

Comment:
I don't think we need `libpq5` or `libpq-dev` anymore FWIW, following a change from @bmwill -- `postgresql` should be enough.